### ISSUE TITLE
tfk.backend.set_floatx('float64') leads to many errors of reload tests.

### ISSUE THREAD

**Issue by nd-02110114** [deepchem/deepchem#2265]
`tfk.backend.set_floatx('float64')`  in `test_normalizing_flow_model_reload` leads to many failures of reload tests.
Please see travis logs from https://travis-ci.org/github/deepchem/deepchem/builds/740412979.
We should fix this issue.

**Comment by nd-02110114** [comment]
`tf.keras.backend.set_floatx('float64')`  in `test_normalizing_flow_model_reload ` cause the unrelated tests' errors. 

Basically, all keras layers return the same numeric type as the input, like if the input is float32, the return is also float32. But, when adding this line,  all keras layers always return the numeric type you set like float64 in spite of the input numeric type. Therefore, the loss function raise some errors because the type is inconsistent between the input and the model's output.

**Comment by ncfrey** [comment]
Should be fixed by #2272

**Comment by rbharath** [comment]
Just merged in #2272. Going to close this issue. Thanks for the bugfind and fix folks!
